src/utilities.js

```diff
@@ -157,3 +157,59 @@ export function createMelFilterBank(numFilters, sampleRate, bufferSize) {
 
   return filterBank;
 }
+
+export function hzToOctaves(freq, A440) {
+  return Math.log2(16 * freq / A440);
+}
+
+export function normalizeByColumn (a) {
+  var emptyRow = a[0].map(() => 0);
```

Comment:
could you use `fill(0)` here?

Reply:
I spent a while scratching my head about why I did this. I finally remembered: I could use `fill` but I'd have to clone `a[0]` so as not to mutate it by which time a `map` is easier.